adam/runtime/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu
+
+MAINTAINER Frank Austin Nothaft, fnothaft@berkeley.edu
+
+# install jdk-7
+RUN apt-get update
+RUN apt-get install -y openjdk-7-jdk
```

Comment:
style per http://docs.docker.com/articles/dockerfile_best-practices/, ```update && install``` should be in the same ```RUN```